src/components/Header/Header.js

```diff
@@ -13,15 +14,75 @@ import {
   Text,
   useColorModeValue,
 } from "@chakra-ui/react";
+import { Select } from "chakra-react-select";
+
+/**
+ * Internal dependencies 
+ */
+import { GetAllOrganizations, getAllEpics } from "../../data/graph-data";
+import { isEmpty } from "../../utils/common";
 
 export default function Header({
+  APIKey,
   onAPIKeyModalOpen = () => {},
   workspace,
   saveWorkspace,
   epic,
   saveEpic,
   epicIssue,
 }) {
+  const [allOrganizations, setAllOrganizations] = useState([]);
+  const [chosenOrganization, setChosenOrganization] = useState(false);
+  const [chosenWorkspace, setChosenWorkspace] = useState(false);
+  const [allEpics, setAllEpics] = useState([]);
+
+  useEffect(() => {
+    if (isEmpty(APIKey)) {
+      return;
+    }
+
+    const controller = new AbortController();
+    const { signal } = controller;
+
+    GetAllOrganizations(
+      "https://api.zenhub.com/public/graphql/",
+      APIKey,
+      signal
+    ).then( ( organizations ) => {
+      setAllOrganizations( organizations );
+    } ).catch((err) => {
```

Comment:
This can be simplified:

```suggestion
    ).then( setAllOrganizations )
    .catch((err) => {
```

Reply:
Updated, thank you!